target/linux/qualcommax/files/arch/arm64/boot/dts/qcom/ipq6018-fap650.dts

```diff
@@ -0,0 +1,458 @@
+			port_id = <3>;
+			phy_address = <2>;
+		};
+
+		port@3 {
+			port_id = <4>;
+			phy_address = <3>;
+		};
+
+		port@4 {
+			port_id = <5>;
+			phy_address = <4>;
+		};
+
+	};
+};
+
+&usb2 {
+	status = "okay";
+};
+
+&usb3 {
+	status = "okay";
+};
+
+&edma {
+	status = "okay";
+};
+
+&dp1 {
+	status = "okay";
+	phy-handle = <&qca8075_0>;
+	label = "lan1";
+};
+
+&dp2 {
+	status = "okay";
+	phy-handle = <&qca8075_1>;
+	label = "lan2";
+};
+
+&dp3 {
+	status = "okay";
+	phy-handle = <&qca8075_2>;
+	label = "lan3";
+};
+
+&dp4 {
+	status = "okay";
+	phy-handle = <&qca8075_3>;
+	label = "lan4";
+};
+
+&dp5 {
+	status = "okay";
+	phy-handle = <&qca8075_4>;
+	label = "wan";
+};
+
+&qpic_bam {
+	status = "okay";
+};
+
+&blsp1_spi1 {
+	status = "okay";
+	
+	partitions {
+		status = "disabled";
+	};
+
+	flash@0 {
+		#address-cells = <1>;
+		#size-cells = <1>;
+		reg = <0>;
+		compatible = "jedec,spi-nor";
+		spi-max-frequency = <50000000>;
+
+		partitions {
+			compatible = "fixed-partitions";
+
+
+			partition@0 {
+				reg = <0x00 0xc0000>;
+				label = "0:sbl1";
+			};
+
+			partition@c0000 {
+				reg = <0xc0000 0x10000>;
+				label = "0:mibib";
+			};
+
+			partition@d0000 {
+				reg = <0xd0000 0x20000>;
+				label = "0:bootconfig";
+			};
+
+			partition@f0000 {
+				reg = <0xf0000 0x20000>;
+				label = "0:bootconfig1";
+			};
+
+			partition@110000 {
+				reg = <0x110000 0x1a0000>;
+				label = "0:qsee";
+			};
+
+			partition@2b0000 {
+				reg = <0x2b0000 0x1a0000>;
+				label = "0:qsee_1";
+			};
+
+			partition@450000 {
+				reg = <0x450000 0x10000>;
+				label = "0:devcfg";
+			};
+
+			partition@460000 {
+				reg = <0x460000 0x10000>;
+				label = "0:devcfg_1";
+			};
+
+			partition@470000 {
+				reg = <0x470000 0x40000>;
+				label = "0:rpm";
+			};
+
+			partition@4b0000 {
+				reg = <0x4b0000 0x40000>;
+				label = "0:rpm_1";
+			};
+
+			partition@4f0000 {
+				reg = <0x4f0000 0x10000>;
+				label = "0:cdt";
+			};
+
+			partition@500000 {
+				reg = <0x500000 0x10000>;
+				label = "0:cdt_1";
+			};
+
+			partition@510000 {
+				reg = <0x510000 0x10000>;
+				label = "0:appsblenv";
+			};
+
+			partition@520000 {
+				reg = <0x520000 0xa0000>;
+				label = "0:appsbl";
+			};
+
+			partition@5c0000 {
+				reg = <0x5c0000 0xa0000>;
+				label = "0:appsbl_1";
+			};
+
+			partition@660000 {
+				reg = <0x660000 0x40000>;
+				label = "0:art";
+			};
+
+		};
+	};
+};
+
+&qpic_nand {
+	status = "okay";
+	partitions {
+		status = "disabled";
+	};
+
+	nand@0 {
+		reg = <0>;
+
+		nand-ecc-strength = <4>;
+		nand-ecc-step-size = <512>;
+		nand-bus-width = <8>;
+	
+
+		partitions {
+			compatible = "fixed-partitions";
+			#address-cells = <1>;
+			#size-cells = <1>;
+
+			partition@0 {
+				label = "rootfs";
+				reg = <0x0 0x3c00000>;
+			};
+			partition@3c00000 {
+				label = "rootfs_1";
+				reg = <0x3c00000 0x3c00000>;
+			};
+		};
+	};
+};
+
+&wifi {
+ 	qcom,ath11k-calibration-variant = "Yuncore-FAP650";
+	qcom,tgt-mem-mode = <1>;
```

Comment:
This doesn't exist

Reply:
What exactly? bdf with this is in this commit.

Reply:
`qcom,tgt-mem-mode` doesnt exist, remove it

Reply:
How to pass 512MB mem mode?

Reply:
see here

https://github.com/openwrt/openwrt/blob/55c7b2cdaf827a4045320f944d73a168cad6bfb7/target/linux/qualcommax/files/arch/arm64/boot/dts/qcom/ipq8070-cax1800.dts#L306

Reply:
removed: qcom,tgt-mem-mode = <1>;


left here: qcom,ath11k-fw-memory-mode = <1>;

Reply:
@kirdesde thanks.